### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by YuriyBilskiy**
[DEMO LINK](https://yuriybilskiy.github.io/react_person/)

### CODE REVIEW

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,10 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ name, age, partner, sex }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+    {age && <p className="Person__age">{`I am ${age}`}</p>}
+    <p className="Person__partner">
+      {`${!partner === true ? 'I am not married' : `${partner} is my
+      ${sex === 'm' ? 'wife' : 'husband'}`}`}
```
Comment (TarasHoliuk) [inline src/components/Person/Person.jsx:7]: that's a bit difficult to read
you may move `${partner} is my ${sex === 'm' ? 'wife' : 'husband'}`}` to a variable above and use here

File: src/App.jsx [hunk 2]
```diff
@@ -25,7 +26,23 @@ export const alex = {
 
 export const App = () => (
   <div className="App">
-    <section className="Person">
+    <Person
+      name={misha.name}
+      age={misha.age}
+      partner={misha.partnerName}
+      sex={misha.sex}
```
Comment (TarasHoliuk) [inline src/App.jsx:33]: do not pass separate properties
you should pass an entire person object as a prop

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
Keep up the good job! Check my comments

**Review by loralevitska — APPROVED** [review]
Good job!

**Pull request closed without merging**